src/Launcher/LauncherBase.php

```diff
@@ -164,18 +118,12 @@ public function run($job) {
         '@error' => $e->getMessage(),
       ));
     }
-    // Restore the user.
-//    $GLOBALS['user'] = $original_user;
-//    drupal_save_session($original_session_saving);
```

Comment:
just because it's commented out doesn't mean that it can be removed ;) That said, UltimateCron might be a better place to implement this. compare it with the parent implementation of cron()